src/main/java/com/seals/camplanner/event/controllers/EventController.java

```diff
@@ -60,6 +62,8 @@ private Event fromDto(EventDto eventDto) {
     }
 
     private List<EventDto> toEventDtoList(List<Event> events) {
-        return events.stream().map(event -> modelMapper.map(event, EventDto.class)).toList();
+        Type type = new TypeReference<List<EventDto>>() {
+        }.getType();
+        return modelMapper.map(events, type);
```

Comment:
There was nothing wrong with the previous way of doing this, I'm just experimenting with [Generics](http://modelmapper.org/user-manual/generics/)